src/partner/modules/EventList/EventListContainer.tsx

```diff
@@ -4,8 +4,9 @@ import { EventList } from "./EventList";
 
 export interface IEventListContainerProps {
   events: IEvent[];
+  handleCancelEvent: Function;
```

Comment:
avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature